explorer/client/src/components/displaybox/DisplayBox.module.css

```diff
@@ -2,8 +2,41 @@ div.imagebox {
     @apply border-0 border-b-2 border-solid border-gray-300;
 }
 
-img.imagebox {
-    @apply object-scale-down w-[100%];
+img.smallimage {
+    @apply object-scale-down w-[100%] cursor-pointer;
+}
+
+div.modalcontainer,
+div.detailsbg {
+    @apply fixed top-0 left-0 w-full h-full z-20;
+}
+
+.modal {
+    @apply w-[100%] h-[90%] max-h-[450px]  z-50 fixed left-0 top-[5%] text-center justify-center flex flex-wrap;
+}
+
+img.largeimage {
+    @apply max-w-[80vw] max-h-[80vh] z-50 self-start border-0;
+
+    border-radius: 6px !important;
+}
+
+span.cross {
+    @apply block text-offwhite bg-sui-grey-100 z-50 h-[30px] w-[30px] self-start
```

Comment:
It seems like we're creating a fair amount of arbitrary values, we can probably move some of these to the tailwind config if there's any consistency in the arbitrary values that are used.

Reply:
I agree.